packages/ui/src/Theme.tsx

```diff
@@ -38,6 +38,12 @@ const defaultTheme: UnifiedTheme = {
   orange: "#e3780c",
   black: "#000000",
 
+  info: "#4a90e2", // blue
```

Comment:
Can you add a story that shows using them?

Reply:
Lemme know if i should add a todo for dark mode logic.

Reply:
Nah that'll be outside this.